### ISSUE TITLE
Correct comment grammar

### ISSUE THREAD

**Issue by mcomella** [mozilla-mobile/focus-android#1885]
[In CustomTabsConfig](https://github.com/mozilla-mobile/focus-android/blob/b80672f06491513eae083e5c9c0c4ba3b68d2cb4/app/src/main/java/org/mozilla/focus/customtabs/CustomTabConfig.java#L166), we have:

```kotlin
// Other than Chrome we will show the share menu by default. Note that currently the
```

This should say,
```kotlin
// Unlike Chrome we will show the share menu by default. Note that currently the
```

Fix it! :)

**Comment by harisankerPradeep** [comment]
I'd like to do this :)

**Comment by mcomella** [comment]
> I'd like to do this :)

FYI: this is PR #1892.

**Comment by mcomella** [comment]
Closed  ab17bff.
